SPRCLI/Package.swift

```diff
@@ -0,0 +1,41 @@
+// swift-tools-version: 5.9
+
+import PackageDescription
+
+let package = Package(
+    name: "SPRCLI",
+    platforms: [
+        .macOS(.v13)
+    ],
+    dependencies: [
+        .package(url: "https://github.com/apple/swift-argument-parser", from: "1.2.0"),
+        .package(url: "https://github.com/awslabs/aws-sdk-swift", from: "0.46.0"),
+        .package(path: "../AWSSDKSwiftCLI"),
```

Comment:
The `AWSCLIUtils` are accessed by adding the local copy of `AWSSDKSwiftCLI` as a dependency.